PythonPonies/bangazon-api
# cleaned up string methods

# Description
1. The string methods for The Payment_Type, Product_On_Order and Products needed updated. Some weren't in place or were returning none for some of the data.
2. I updated the readme to include information using the superuser

## Number of Fixes
2

## Related Ticket(s)
#78 #69

## Problem to Solve
The string methods for The Payment_Type, Product_On_Order and Products needed updated. Some weren't in place or were returning none for some of the data.

The readme also needed more information.

## Proposed Changes
I updated string methods for these three classes and updated the readme.

## Expected Behavior
You can view these models in the admin interface and see descriptions rather than just seeing the word `object` and you should see superuser information in the readme.

## Steps to Test Solution

1. Git fetch this branch
1. Log into http://127.0.0.1:8000/admin
1. Navigate to each object type to see if a description appears.

Then view the readme add review changes.

## Testing

[ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[ ] I certify that all existing tests pass

## Documentation
[X] There is new documentation in this pull request that must be reviewed.
[ ] I added documentation for any new classes/methods

## Deploy Notes
None
